client/components/web-preview/toolbar.jsx

```diff
@@ -207,7 +207,12 @@ export default connect(
 
 		return {
 			canUserEditThemeOptions,
-			customizeUrl: getCustomizerUrl( state, siteId ),
+			customizeUrl: getCustomizerUrl(
+				state,
+				siteId,
+				null,
+				window.location.pathname + window.location.search
```

Comment:
For this to work in Jetpack sites, I think this one will need to be:

```
window.location.href + window.location.search
```

Otherwise the `returnUrl` will be to a path within the Jetpack site itself, instead of back to Calypso.

That said, it doesn't look like the `return` feature is working properly on `master` for Jetpack sites in my testing (the close button just takes me to the root of the site), and fixing that behaviour is probably out of scope for this PR. But for the moment, if we use `window.location.href`, at least the return url string will be consistent.

Reply:
@andrewserong Thanks for testing

I was afraid this was too easy. 🍭 
I used the pathname because we add the host again to the url:

https://github.com/Automattic/wp-calypso/blob/b11c9c2/client/my-sites/customize/main.jsx#L191

I think maybe I'll create an exception if we find a redirect param.

Reply:
Updated! Using `window.location.href` since it includes the query string